inc/parser/xhtml.php

```diff
@@ -96,6 +96,10 @@ public function startSectionEdit($start, $data) {
      */
     public function finishSectionEdit($end = null, $hid = null) {
         $data = array_pop($this->sectionedits);
+        if(is_null($data)) {
```

Comment:
wouldn't a count() before the pop make more sense?

Reply:
You're right of course - I'll update it